network/p2p/peer_tracker.go

```diff
@@ -17,10 +17,9 @@ import (
 	"github.com/ava-labs/avalanchego/utils"
 	"github.com/ava-labs/avalanchego/utils/heap"
 	"github.com/ava-labs/avalanchego/utils/logging"
+	safemath "github.com/ava-labs/avalanchego/utils/math"
```

Comment:
I think we try avoiding safemath alias if we don't use math package from standard library

Reply:
the `math` package is imported. Can we move this back to where it was previously though?

Reply:
We can move this.... is this our import style to put all aliased imports in their own block?